documentation/modules/exploit/windows/http/ektron_xslt_exec_ws.md

```diff
@@ -0,0 +1,70 @@
+## Vulnerable Application
+
+  Ektron Content Management System (CMS) 8.5 and 8.7 before 8.7sp2 and 9.0 before sp1 (according to CVE-2015-0923)
```

Comment:
`[Ektron Content Management System (CMS)](http://www.episerver.com/cms/ektron/)`
I dont see a way to download a vuln copy (looks like you have to pay)
`[CVE-2015-0923](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-0923)`